src/platform/Tizen/DnssdImpl.cpp

```diff
@@ -352,6 +354,14 @@ void OnResolve(dnssd_error_e result, dnssd_service_h service, void * data)
 
     dnssdService.mPort = static_cast<uint16_t>(port);
 
+    ret = dnssd_service_get_interface(service, &interface);
+    VerifyOrExit(ret == DNSSD_ERROR_NONE, ChipLogError(DeviceLayer, "dnssd_service_get_interface() failed. ret: %d", ret));
+
+    err = chip::Inet::InterfaceId::InterfaceNameToId(interface, dnssdService.mInterface);
+    VerifyOrExit(
+        err == CHIP_NO_ERROR,
+        ChipLogError(DeviceLayer, "chip::Inet::InterfaceId::InterfaceNameToId() failed. ret: %" CHIP_ERROR_FORMAT, err.Format()));
```

Comment:
This path is passing `ret`==`DNSSD_ERROR_NONE` to `rCtx->mCallback(rCtx->mCbContext, nullptr, chip::Span<chip::Inet::IPAddress>(), GetChipError(ret));` at the `exit:` label.

Reply:
Good catch! I've missed that...

Reply:
Thank you for your detail review! I updated it.